config/locales/simple_form.en.yml

```diff
@@ -1,4 +1,8 @@
 en:
+  helpers:
+    submit:
+      create: "Create"
```

Comment:
In my head it made more sense as you are 'adding' something to the existing set of 'xyz'. However, after doing some research I will put it down as a language thing.

http://ux.stackexchange.com/questions/53270/what-text-should-be-used-for-an-insertion-button-create-new-or-add